packages/components/package.json

```diff
@@ -29,10 +29,13 @@
 		"@babel/runtime": "^7.13.10",
 		"@emotion/core": "^10.1.1",
 		"@emotion/css": "^10.0.22",
+		"@emotion/hash": "^0.8.0",
+		"@emotion/is-prop-valid": "^0.8.8",
 		"@emotion/native": "^10.0.22",
 		"@emotion/styled": "^10.0.23",
 		"@wordpress/a11y": "file:../a11y",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/data": "file:../data",
```

Comment:
Ideally, the components package shouldn't rely on the data packages. What's the reason for this?